packages/create-gatsby/src/index.ts

```diff
@@ -44,6 +45,42 @@ const questions = [
   },
 ]
 
+const supportedOptionTypes = [`string`, `boolean`, `number`]
+
+const makePluginConfigQuestions = (
+  selectedPlugins: Array<string>
+): Array<any> => {
+  const formPrompts = selectedPlugins.map((pluginName: string) => {
+    const schema = pluginSchemas[pluginName]
+    const { keys: options } = schema
+    const choices = Object.keys(schema?.keys).reduce((result, name) => {
+      const option = options[name]
+
+      if (option?.flags?.presence === `required`) {
```

Comment:
Another readability suggestion: you could reverse the check here and do an early return, instead of wrapping the whole lot in the conditional